### DISCUSSION TITLE
Printer ink?

### DISCUSSION THREAD

**Discussion by jaredcat** [andrew-codechimp/HA-Battery-Notes#1896] [Ideas]
I had an idea to use this to track printer ink/toner.
Would I just have to provide a template to get it to work?

Any thoughts on this?

**Comment by andrew-codechimp** [comment]
Interesting, I don't think Battery Notes would fit for this but I did wonder about a more general "Maintenance Notes" as a separate integration.
I presume you were looking at the idea of the Last Replaced to track how long your ink lasts or is there anything more?

**Reply by jaredcat** [reply]
I want to trigger alerts to replace or order more toner when low.  
Basically, exactly the same as a battery in almost every way.
I was starting to make my own automation and was like "i wish battery notes would just take this printer in and do all the stuff it already does"

Heres my printer for example:
![image](https://github.com/user-attachments/assets/6da51078-4b93-48a9-b579-e103cd282cba)

**Reply by andrew-codechimp** [reply]
You could do it with battery notes entity sensors and a template for the low warning. 
For 1 printer in your setup I'd opt for an automation with a trigger on each of the percentages and then work out which one triggered it and fire off your notifications, battery notes is really more for bulk devices where writing an automation to handle so many would be painful.
